inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/settings/WorkloadPanel.java

```diff
@@ -96,29 +88,16 @@ public WorkflowAndMonitoringPanel(String aID, IModel<Project> aProject) {
         add(form);
     }
 
-    @Override
-    protected void onInitialize()
-    {
-        super.onInitialize();
-        IModel<String> resourceModel = new StringResourceModel(
-            getString(getId()), this, Model.of(getId()));
-    }
-
-    @Deprecated
     private void actionConfirm(AjaxRequestTarget aTarget, Form<?> aForm)
     {
         aTarget.addChildren(getPage(), IFeedback.class);
 
-        if (monitoringChoices.getDefaultModelObjectAsString().equals(monitoring.get(0))) {
-            workloadProperties.setActive(false);
-        } else {
-            workloadProperties.setActive(true);
-        }
-
-        if (workflowChoices.getDefaultModelObjectAsString().equals(workflow.get(0))) {
-            workflowProperties.setActive(false);
+        if (monitoringChoices.getDefaultModelObjectAsString().equals(getString("workload"))) {
```

Comment:
This way of obtaining the extensions goes away once the workload selection widget is constructed properly - see previous comment.